src/items/functions.md

```diff
@@ -294,9 +294,10 @@ from the "Native unwind" column in the table.
 | `panic=abort`  | unwinding     | `panic` aborts without unwinding      | abort                    |
 | `panic=abort`  | non-unwinding | `panic` aborts without unwinding      | [undefined behavior]     |
 
-> **Note**: With `panic=unwind`, when a `panic` is turned into an abort by a
-> non-unwinding ABI boundary, either no destructors (`Drop` calls) will run, or
-> all destructors up until the ABI boundary will run.
+> **Note**: The following guarantee applies from Rust 1.82 onward: with
+> `panic=unwind`, when a `panic` is turned into an abort by a non-unwinding
+> ABI boundary, either no destructors (`Drop` calls) will run, or all destructors
+> up until the ABI boundary will run.
```

Comment:
Isn't this a normative guarantee? Why is it a "note"?

Reply:
The version should be in a note, but yeah the rest of it should be in normative text.

Reply:
I won't have a computer for this entire week, so if either of you can make this change, I'd appreciate it.

(It was a note because, when this was originally added, I wasn't aware of a rule that all notes were non-normative.)